Webpack compilation fails because of missing 'mkdirp' with latest npm

```
2021-08-23 14:57:52.239  INFO 85514 --- [nPool-worker-19] c.v.base.devserver.DevModeHandlerImpl    : Running webpack to compile frontend resources. This may take a moment, please stand by...
node:internal/modules/cjs/loader:943
  throw err;
  ^

Error: Cannot find module 'mkdirp'
```

npm version 7.20.6

Looks like `target/plugins/stats-plugin/stats-plugin.js` depends on `mkdirp` but the project has no dependency for it

--- a/flow-server/src/main/resources/plugins/application-theme-plugin/package.json
+++ b/flow-server/src/main/resources/plugins/application-theme-plugin/package.json
@@ -6,12 +6,16 @@
   ],
   "repository": "vaadin/flow",
   "name": "@vaadin/application-theme-plugin",
-  "version": "0.4.0",
+  "version": "0.4.1",
   "main": "application-theme-plugin.js",
   "author": "Vaadin Ltd",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/vaadin/flow/issues"
+  },
+  "dependencies": {
+    "mkdirp": "0.5.5",
+    "glob": "7.1.6"
   },
   "files": [
     "application-theme-plugin.js",

--- a/flow-server/src/main/resources/plugins/stats-plugin/package.json
+++ b/flow-server/src/main/resources/plugins/stats-plugin/package.json
@@ -13,7 +13,7 @@
     "url": "https://github.com/vaadin/flow/issues"
   },
    "dependencies": {
-    "mkdirp": "0.5.5",
+    "mkdirp": "0.5.5"
   },
   "files": [
     "stats-plugin.js"

--- a/flow-server/src/main/resources/plugins/stats-plugin/package.json
+++ b/flow-server/src/main/resources/plugins/stats-plugin/package.json
@@ -12,6 +12,9 @@
   "bugs": {
     "url": "https://github.com/vaadin/flow/issues"
   },
+   "dependencies": {
+    "mkdirp": "0.5.5",
+  },
   "files": [
     "stats-plugin.js"
   ]

